build/int.cloudbuild.yaml

```
@@ -128,7 +128,7 @@ tags:
 - 'integration'
 substitutions:
   _DOCKER_IMAGE_DEVELOPER_TOOLS: 'cft/developer-tools'
-  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.2.3'
+  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.13.9'
```

```suggestion
  _DOCKER_TAG_VERSION_DEVELOPER_TOOLS: '1.13'
```
so that we can have the patch version updated